script/sliding_window.py

```diff
@@ -59,4 +63,4 @@
 
 plt.savefig(fn, dpi=300)
 
-#plt.show()
+plt.show()
```

Comment:
Unless we set the backend to template or Agg during build, this will block the Makefile execution.